docs/blog/20230501-Daeraxa-MayUpdate.md

```diff
@@ -0,0 +1,168 @@
+
+## CI Testing speed upgrade
+
+<img src="./assets/ghtests.png"/>
+
+[@DeedeeG] has [made an update](https://github.com/pulsar-edit/pulsar/pull/492) to our continuous integration configuration by caching and restoring the dependencies which ends up saving us a huge amount of time, possibly over an hour each run! (Pulsar has to build and test an awful lot of packages on each run so it is a long process). This should hopefully make life just that bit easier on the development side of things.
+
+## Backend webhook
+
+<img src="./assets/package-webhook.png"/>
+
+[@confused-techie] has been busy adding yet another [new feature](https://github.com/pulsar-edit/package-backend/pull/156) to our package backend, this time it is a new webhook that allows us to publish messages and notifications when package authors publish and update their packages. You can check this out in action in our [backend-notifications](https://discordapp.com/channels/992103415163396136/1045131651748999239) channel on Discord.
+
+## Package website readme links overhaul
+
+<img src="./assets/package-frontend.png"/>
+
+Now some [changes to our package website](https://github.com/pulsar-edit/package-frontend/pull/96). There are a few separate, but related, changes by [@confused-techie] here that involve improving our package website to overhaul some of the issues we have with readme links.
+
+The first change adds shorthand string author assignment in the `package.json` file of a package. The [npm package specification(https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors)] allows the three author properties to be combined into a single string e.g.
+
+```js
+{
+  "author": "Barney Rubble <b@rubble.com> (http://barnyrubble.tumblr.com/)"
+}
+```
+
+Previously if this was not populated fully with all three options then the author simply wouldn't display. This fixes that problem by allowing the optional fields to be omitted (actually this is taken a step further in our implementation - even `name` can be optional)
+
+In a similar vein we also now support [shortcut syntax](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository) for the `repository` field of the `package.json`. For example writing `authorName/repoName` wasn't working so we have provided support for this.
+
+We noticed that many package authors would link to other community packages in their readmes in order to show other packages that work with or complement their package. Unfortunately as these point to the old `atom.io` website it means that every single link simply redirects to the [sunsetting Atom](https://github.blog/2022-06-08-sunsetting-atom/) page. This is really not helpful as it entirely breaks the embedded links, sure there are workaround, people could look at the link and work out what packages it links to but this is a hassle when all you want to do is click a link.
+
+<img src="./assets/sunsetting.png"/>
```

Comment:
For the future, it's best to either omit the trailing slash here…

```html
<img src="./assets/sunsetting.png">
```

…or add a space before it.

```
<img src="./assets/sunsetting.png" />
```

I don't imagine anything will explode, but the way you've got it here is not valid HTML.

Reply:
Oh that was my mistake doing some mass updates to remove the height option, should probably fix that, the space was there before.